cmd/openshift-install/agent/waitfor.go

```diff
@@ -85,9 +89,11 @@ func newWaitForInstallCompleteCmd() *cobra.Command {
 				logrus.Exit(exitCodeBootstrapFailed)
 			}
 
+			if err := agentpkg.WaitForBootstrapComplete(cluster); err != nil {
+				handleBootstrapError(cluster, err)
+			}
+
 			if err = agentpkg.WaitForInstallComplete(cluster); err != nil {
-				logrus.Debug("Printing the event list gathered from the Agent Rest API")
-				cluster.PrintInfraEnvRestAPIEventList()
```

Comment:
I'm curious why we would stop printing out the event list if the install failed to complete. Wouldn't it be useful information?

Reply:
The theory is that at this point we've succeeded in bootstrapping, and it's just that the CVO isn't complete, so the events that happened before/during bootstrapping seem less relevant at this point.
I think this was previously here because it was trying to cover all cases, since there was previously no way of telling if it failed during bootstrapping or after.